TELCODOCS-628: Disabling chronyd in PCG CR (#48933)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.

[id="ocp-4-11-RAN-disabling-chronyd"]
====  Disable chronyd in the PolicyGenTemplate CR

> You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:

Reviewer: To set context for the user, it might help to add a reference to the [section in the main docs](http://file.emea.redhat.com/amolnar/TELCODOCS-628/scalability_and_performance/ztp-vdu-validating-cluster-tuning.html#recommended-cluster-kernel-configuration), where this configuration is being discussed.

[source,yaml]
----
[service]

> service.stalld=start,enable

Reviewer: Seems like the line that is actually disabling `chronyd` is [this](https://github.com/openshift/openshift-docs/pull/48933/files#diff-c77c0cbc134fbd6d7bfb05509f65f4077fbbe88a6ce62ddafb44edc35e479f26R1062). Unless they are related, I suggest removing the following line. I also noticed that the following config is already mentioned in the released version.
```suggestion

```

Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.

[id="ocp-4-11-RAN-disabling-chronyd"]
====  Disable chronyd in the PolicyGenTemplate CR

> You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the source CRs. Add the following line in `TunedPerformancePatch.yaml`:

Reviewer: ```suggestion
You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the ztp-site-generate container image:
```

> You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the source CRs. Add the following line in `TunedPerformancePatch.yaml`:

Reviewer: ```suggestion
You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the `ztp-site-generate` container image:
```

[id="ocp-4-11-RAN-disabling-chronyd"]
====  Disable chronyd in the PolicyGenTemplate CR

> You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the `ztp-site-generate` container image:

Reviewer 2: This is referring to the PolicyGenTemplate CR in the group policies (vs source-crs). 
    - fileName: TunedPerformancePatch.yaml
      policyName: "config-policy"
      spec:
        profile:
          - name: performance-patch
            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
            data: |
              [main]
              summary=Configuration changes profile inherited from performance created tuned
              include=openshift-node-performance-openshift-node-performance-profile
              [bootloader]
              cmdline_crash=nohz_full=2-23,26-47
              [sysctl]
              kernel.timer_migration=1
              [service]
              service.stalld=start,enable
              service.chronyd=stop,disable
